src/OnlineSales/Migrations/ApiDbContextModelSnapshot.cs

```diff
@@ -1154,6 +1161,18 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Navigation("Post");
                 });
 
+            modelBuilder.Entity("OnlineSales.Entities.Contact", b =>
+                {
+                    b.HasOne("OnlineSales.Entities.Domain", "Domain")
+                        .WithMany()
+                        .HasForeignKey("DomainId")
+                        .OnDelete(DeleteBehavior.Cascade)
```

Comment:
@WASandaruwan am I right that if we delete a domain it will basically delete all the corresponding Contacts? If so, that is not good. It should be  impossible to delete a domain if it has any dependencies in contact table.